### ISSUE TITLE
Create a DTS Projects Listing

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/transportation.austintexas.io#280]
We need a public listing of our current active projects somewhere on transportation.austintexas.io. Could be a dedicated page, or a section of our "About" page.

The project information could ideally be pulled directly from our Github issues. This is contingent on us adopting the practice of creating project-level issues in the `atd-data-tech` repo.

**Comment by mateoclarke** [comment]
Kinda related, maybe a duplicate of this: https://github.com/cityofaustin/transportation.austintexas.io/issues/245

**Comment by johnclary** [comment]
D'oh. Def a duplicate. Time is a flat circle.

**Comment by johnclary** [comment]
Deleting this duplicate issue.
